Dashboard/src/style.css

```
@@ -8,6 +8,9 @@
 }
 
 body {
-  font-family: sans-serif;
+  font-family: "Inter", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI",
```

Stylelint (17.11.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)